TestSuite/test_suite_base.py

```diff
@@ -0,0 +1,31 @@
+from abc import abstractmethod
+from pathlib import Path
+
+from demisto_sdk.commands.content_graph.interface import ContentGraphInterface
+from demisto_sdk.commands.content_graph.objects.base_content import BaseContent
+
+
+class TestSuiteBase:
+    def __init__(self, path: Path):
+        self.obj_path = path
+
+    @property
+    def object(self):
+        return BaseContent.from_path(self.obj_path)
+
+    def get_graph_object(self, interface: ContentGraphInterface):
+        return interface.from_path(self.obj_path)
```

Comment:
please document the difference. Document that this is with `relationships_data` populated.